### PR TITLE
Adds S3ObjectStore import to util __init__.py 

### PR DESCRIPTION
**Pull request by codestar12**
Adds S3ObjectStore import to util __init__.py so that S3ObjectStore closure is constant with LibCloudObjectStore

### CODE REVIEW

File: composer/utils/__init__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     'ObjectStore',
     'ObjectStoreTransientError',
     'LibcloudObjectStore',
+    'S3ObjectStore'
     'MissingConditionalImportError',
```
Comment (hanlint) [inline composer/utils/__init__.py:27]: Could you add `SFTPObjectStore` as well? 🙏

**Review by hanlint — APPROVED** [review]

**Review by ravi-mosaicml — APPROVED** [review]
LGTM, thanks!

File: composer/utils/__init__.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
     'ObjectStore',
     'ObjectStoreTransientError',
     'LibcloudObjectStore',
+    'S3ObjectStore'
```
Comment (hanlint) [inline composer/utils/__init__.py:26]:
```suggestion
    'S3ObjectStore',
```

File: composer/utils/__init__.py (hunk 1 above)
Comment (codestar12) [reply composer/utils/__init__.py:28]: done

File: composer/utils/__init__.py (hunk 2 above)
Comment (codestar12) [reply composer/utils/__init__.py]: nice catch. Fixed when adding SFTPObjectStore

**Pull request merged**